dev_guide/service_accounts.adoc

```diff
@@ -213,6 +213,35 @@ Tokens:            	robot-token-8bhpp
 ----
 ====
 
+== Checking if Pod Creation is Possible
+
+Using the `scc-review` and `scc-subject-review` options, you can check if a user, service account, or group can create a pod. The command outputs the security context constraints that admit the resource.
+
+To check if a specific service account can create a pod in a specific file:
+
+----
+$ oc policy scc-review -z <service-account1>,<service_account2> -f myfile.yaml
+----
+
+To check if a user with the system:serviceaccount:username:default service account can a create a pod:
```

Comment:
The 3rd part is a project name, not a username:
`s/system:serviceaccount:username:default/system:serviceaccount:projectname:default/`